services/identity/ReposService.ts

```diff
@@ -76,8 +78,18 @@ export default class ReposService {
       description: `Staging: ${stagingUrl} | Production: ${productionUrl}`,
     })
 
-    const dir = `/tmp/${repoName}`
-    const remote = "origin"
+    const dir = this.localRepoPath(repoName)
+
+    // 1. Set URLs in local _config.yml
+    const configPath = `${dir}/_config.yml`
+    const configFile = fs.readFileSync(configPath, "utf-8")
+    // Assume the last two lines of config contain outdated staging and prod urls
+    const lines = configFile.split("\n").slice(0, -2)
+    lines.push(`staging: ${stagingUrl}`)
+    lines.push(`prod: ${productionUrl}`)
+    fs.writeFileSync(configPath, lines.join("\n"))
```

Comment:
If the last two lines are not useful, we could modify the site creation to remove those lines instead?

Reply:
I'd like to, but we'd have to find another way to add them for Netlify sites. (We could give up on this and officially turn away from Netlify, but I have so far managed to keep our existing process running exactly as it was.)

Reply:
Now that I'm thinking it through, we could get away with doing this, as these two lines are currently useless. However, it might be good to set up staging and prod deployments for this base repo and change the URLs instead of getting rid of them. If we did this, then the base repo would be editable like any other isomer site. That would be a nice feature.

Reply:
Ok, my latest changes allow the staging and prod URLs to be anywhere in `_config.yml` and throw an error if either one is not present.